apps/web/src/services/routes.ts

```diff
@@ -5,6 +5,7 @@ const PROJECTS_PATH = `${ROOT_PATH}projects`
 
 export const ROUTES = {
   root: ROOT_PATH,
+  settings: '/settings',
```

Comment:
Do an object please and name this `root`. This way later if we want to add more sub routes is easier